src/main/java/com/google/flourbot/execution/MacroExecutionModule.java

```diff
@@ -3,5 +3,6 @@
 import java.util.concurrent.ExecutionException;
 
 public interface MacroExecutionModule {
-    public String execute(String userEmail, String message) throws IllegalStateException, InterruptedException, ExecutionException;
-}
\ No newline at end of file
+  String execute(String userEmail, String message)
+      throws IllegalStateException, InterruptedException, ExecutionException;
```

Comment:
no need to declare IllegalStateException (unchecked) and the other I'm guessing can be converted to unchecked as well.

Reply:
Converted checked to unchecked exceptions in #16